content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst

```diff
@@ -0,0 +1,244 @@
+=====
+Egypt
+=====
+
+.. _egypt/installation:
+
+Installation
+============
+
+:ref:`Install <general/install>` the following modules to get all the features of the Egyptian
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Egypt - Accounting`
+     - ``l10n_eg``
+     - Default :doc:`fiscal localization package <../overview/fiscal_localization_packages>`
+   * - :guilabel:`Egyptian E-invoice Integration`
+     - ``l10n_eg_edi_eta``
+     - :ref:`Egyptian Tax Authority (ETA) e-invoicing integration <egypt/e-invoicing>`
+
+.. _egypt/e-invoicing:
+
+Egyptian e-invoicing
+====================
+
+Odoo is compliant with the **Egyptian Tax Authority (ETA) e-invoicing** requirements.
+
+.. important::
+   Egyptian e-invoicing is available from Odoo 15.0. If needed, :doc:`upgrade
+   </administration/upgrade>` your database.
+
+.. seealso::
+   - `Video: Egypt E-invoicing <https://www.youtube.com/watch?v=NXuBPLR4pVw>`_
```

Comment:
Is it intentional not to embed the video?

Reply:
It is indeed. I don't want the video to become the primary info here. But it is listed at the top of the page to make it easily reachable.